.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,78 @@
+name: Integration test with mediamtx stream
+
+on: [push]
+
+jobs:
+  build-and-test:
```

Comment:
Might be a good idea to set up a `matrix` here to run build/test on both x86 and arm64.

Reply:
I don't think arm64 runners are working properly... https://github.com/hexbabe/sean-viamrtsp/actions/runs/8484610925/job/23247871489?pr=1 <- this run should fail because I forgot to update the binaries (viam-server and mediamtx) to be for `arm64`, yet the CI still passed. I think in viam-robotics we use buildjet to do arm64 CI, we don't have that here